frontend/src/sdk/commonFetch.ts

```diff
@@ -63,13 +63,17 @@ export const commonFetch =
       },
     };
 
+    const prefix = pathPrefix ?? k8sBasePath;
+
     try {
       let safeURL = url;
       if (/^\/\//.test(url)) {
         // https://github.com/openshift/dynamic-plugin-sdk/pull/55
         safeURL = url.slice(1);
       }
-      return validateStatus(await fetch(new Request(`${k8sBasePath}${safeURL}`, { credentials: 'include' }), allOptions));
+      return validateStatus(
+        await fetch(new Request(`${`${prefix.indexOf('/') === 0 ? '' : '/'}${prefix}`}${safeURL}`, { credentials: 'include' }), allOptions),
```

Comment:
@karelhala small NIT - but can we perform the leading `/` check on line 66 where we initialize the prefix for readability?

Reply:
Sure thing!